sdk/cosmos/test-resources.json

```diff
@@ -16,12 +16,42 @@
       "metadata": {
         "description": "Default Cosmosdb Account level consistency"
       }
+    },
+    "enableMultipleRegions": {
+      "type": "bool",
+      "metadata": {
+        "description": "Enable multiple regions,d efault is false"
```

Comment:
typo

```suggestion
        "description": "Enable multiple regions, default is false"

```

Reply:
@moderakh  - we are still experimenting with this feature. 
This was just a quick commit I asked @danieljurek  to do so I can pick up the changes we talked about :) 
This will all be fixed with better comments.